.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*       @vdomnich-yadro @dansingjulia @yadro-vavdeev
+*       oleg@nspcc.ru vvarg229@gmail.com
```

Comment:
Better have GH user names here. Add @roman-khimov and @cthulhu-rider as well.

Reply:
Done